main.go

```diff
@@ -125,6 +127,9 @@ func main() {
 	if errorOnReplace {
 		opts = append(opts, injectproxy.WithErrorOnReplace())
 	}
+	if regexMatch {
+		opts = append(opts, injectproxy.WithRegexMatch())
```

Comment:
in case of static label value (e.g. `-label-value`), we can fail error if several values are provided.

Reply:
Done.